core-primitives/top-pool/Cargo.toml

```diff
@@ -61,7 +61,7 @@ byteorder = { version = "1.4.2", default-features = false }
 codec  = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
 derive_more = { version = "0.99.5" }
 log = { version = "0.4", default-features = false }
-retain_mut = { version = "0.1.2"}
+retain_mut = { version = "0.1.4"}
```

Comment:
Clippy suddenly went off about `RetainMut` being deprecated. Updated the library, and it seems to be fine now.

Either way, because of that Clippy lint I noticed two `unwrap()` below, and removed them accordingly.